onnxruntime/core/providers/shared/utils/utils.cc

```diff
@@ -66,7 +66,13 @@ bool GetClipMinMax(const InitializedTensorSet& initializers, const Node& node,
         LOGS(logger, VERBOSE) << "Input min of Clip must be known";
         return false;
       }
-      min = GetTensorFloatData(*initializers.at(min_name))[0];
+      std::vector<uint8_t> unpacked_tensor;
+      auto status = onnxruntime::utils::UnpackInitializerData(*initializers.at(min_name), unpacked_tensor);
+      if (!status.IsOK()) {
+        LOGS(logger, ERROR) << "Error while unpack min tensor: " << status.ErrorMessage();
+        return false;
+      }
+      min = reinterpret_cast<float*>(unpacked_tensor.data())[0];
```

Comment:
is this `vector<uint8_t>::data` guaranteed to be suitably aligned for floats?

Reply:
In theory the data in the vector will be aligned to std::max_align_t, which is usually 16, 
To make it more robust, we can change the `unpacked_tensor.resize(tensor_byte_size);` to something like
`unpacked_tensor = std::vector<std::byte>(tensor_byte_size, custom_alligned_allocator);`

Reply:
thanks - sounds like it should be fine for now then